src/Decompiler/Evaluation/ExpressionSimplifier.cs

```diff
@@ -389,6 +391,18 @@ public virtual Expression VisitBinaryExpression(BinaryExpression binExp)
                 return shiftShift.Transform();
             }
 
+            if (complicatedNot.Match(binExp))
+            {
+                Changed = true;
+                return complicatedNot.Transform();
+            }
+
+            if (complicatedNot.Match(binExp))
+            {
+                Changed = true;
+                return complicatedNot.Transform();
+            }
+
```

Comment:
Duplicated call of `complicatedNot` rule

Reply:
Thanks, I noticed that too, it's fixed on the upcoming push :)